routing/payment_session_test.go

```diff
@@ -8,6 +8,54 @@ import (
 	"github.com/lightningnetwork/lnd/routing/route"
 )
 
+func TestValidateCLTVLimit(t *testing.T) {
+	t.Parallel()
+
+	testCases := []struct {
+		cltvLimit      uint32
+		finalCltvDelta uint16
+		includePadding bool
+		expectError    bool
+	}{
+		{
+			cltvLimit:      uint32(103),
+			finalCltvDelta: uint16(100),
+			includePadding: true,
+			expectError:    true,
+		},
+		{
+			cltvLimit:      uint32(104),
+			finalCltvDelta: uint16(100),
+			includePadding: true,
+			expectError:    false,
+		},
+		{
+			cltvLimit:      uint32(100),
+			finalCltvDelta: uint16(100),
+			includePadding: false,
+			expectError:    true,
+		},
+		{
+			cltvLimit:      uint32(101),
+			finalCltvDelta: uint16(100),
+			includePadding: false,
+			expectError:    false,
+		},
+	}
+
+	for _, testCase := range testCases {
```

Comment:
if it's meant to use [table drive test](https://github.com/golang/go/wiki/TableDrivenTests), you'll want sth like,
```
tc := testCase
t.Run(tt.name, func(t *testing.T) {
    ...
}
```

Reply:
thanks, forgot to do it like this